src/client/Microsoft.Identity.Client/IPublicClientApplication.cs

```diff
@@ -159,5 +159,11 @@ public partial interface IPublicClientApplication : IClientApplicationBase
             IEnumerable<string> scopes,
             string username,
             string password);
+
+        /// <summary>
+        /// Used to determine if the currently available broker is able to perform Proof-of-Possesion.
+        /// </summary>
+        /// <returns>Boolean indicating Proof-of-Possesion is supported</returns>
+        public bool IsProofOfPosessionSupportedByClient();
```

Comment:
1. Please update existing integration tests to use this helper? 
2. Please update the docs of WithProofOfPossession to mention that is should be used in conjunction with this helper